package.json

```diff
@@ -4,34 +4,34 @@
   "description": "New Relic Documentation",
   "version": "0.1.0",
   "dependencies": {
-    "@emotion/core": "^10.1.1",
-    "@emotion/styled": "^10.0.27",
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.3.0",
     "@github-docs/frontmatter": "^1.3.1",
     "@mdx-js/mdx": "^2.0.0-next.8",
     "@mdx-js/react": "^2.0.0-next.8",
-    "@newrelic/gatsby-theme-newrelic": "^1.38.0",
+    "@newrelic/gatsby-theme-newrelic": "^2.0.0",
     "@splitsoftware/splitio-react": "^1.2.4",
     "babel-jest": "^26.3.0",
     "common-tags": "^1.8.0",
     "date-fns": "^2.17.0",
     "front-matter": "^4.0.2",
-    "gatsby": "^2.32.4",
-    "gatsby-image": "^2.11.0",
-    "gatsby-plugin-manifest": "^2.12.0",
+    "gatsby": "^3.3.1",
+    "gatsby-image": "^3.3.0",
+    "gatsby-plugin-manifest": "^3.3.0",
     "gatsby-plugin-json-output": "^1.2.0",
     "gatsby-plugin-mdx": "^1.10.1",
     "gatsby-plugin-meta-redirect": "^1.1.1",
     "gatsby-plugin-offline": "^3.10.1",
```

Comment:
This has a newer version as well, `4.3.0`

Reply:
updated.